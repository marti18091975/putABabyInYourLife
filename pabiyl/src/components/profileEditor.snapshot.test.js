import renderer from 'react-test-renderer';
import React from 'react';
import ProfileEditor from './ProfileEditor';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App snapshot', () => {

    const tree = renderer.create(
        <Router>
            <ProfileEditor />
        </Router>
    );

    it('should match', () => {
        expect(tree.toJSON()).toMatchSnapshot();
    });
});