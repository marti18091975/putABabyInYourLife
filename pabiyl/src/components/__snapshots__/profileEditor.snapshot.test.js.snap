// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<main
  className="main__detail"
>
  <form
    id="profileForm"
    method="get"
  >
    <div
      className="left-side"
    >
      <section
        className="container__info--profile"
      >
        <div
          className="container"
        >
          <input
            className="input__name text__detail--profile"
            onChange={[Function]}
            placeholder="name"
            type="text"
          />
          <div
            className="container__detail"
          >
            <img
              alt="gender"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/3260/3260246.svg"
            />
            <input
              className="checkbox"
              name="gender"
              onChange={[Function]}
              type="radio"
              value="male"
            />
            <div
              className="work__filter--text filter__text"
            >
              male
            </div>
            <input
              className="checkbox"
              name="gender"
              onChange={[Function]}
              type="radio"
              value="female"
            />
            <div
              className="work__filter--text filter__text"
            >
              female
            </div>
          </div>
          <div
            className="container__detail"
          >
            <img
              alt="age"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/3021/3021790.svg"
            />
            <input
              className="text__detail--profile"
              onChange={[Function]}
              placeholder="age"
              type="text"
            />
          </div>
          <div
            className="container__detail"
          >
            <img
              alt="City"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/888/888063.svg"
            />
            <input
              className="text__detail--profile"
              onChange={[Function]}
              placeholder="city"
              type="text"
            />
          </div>
        </div>
        <div
          className="container"
        >
          <div
            className="container__detail"
          >
            <img
              alt="country"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/3455/3455212.svg"
            />
            <input
              className="text__detail--profile"
              onChange={[Function]}
              placeholder="country"
              type="text"
            />
          </div>
          <div
            className="container__detail"
          >
            <img
              alt="civil state"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/1102/1102457.svg"
            />
            <div
              className="civil__chekbox--container"
            >
              <div
                className="civil__chekbox--container-side"
              >
                <input
                  className="checkbox"
                  name="civilState"
                  onChange={[Function]}
                  type="radio"
                  value="single"
                />
                <div
                  className="work__filter--text filter__text"
                >
                  single
                </div>
                <input
                  className="checkbox"
                  name="civilState"
                  onChange={[Function]}
                  type="radio"
                  value="divorced"
                />
                <div
                  className="work__filter--text filter__text"
                >
                  divorced
                </div>
              </div>
              <div
                className="civil__chekbox--container-side"
              >
                <input
                  className="checkbox"
                  name="civilState"
                  onChange={[Function]}
                  type="radio"
                  value="widow"
                />
                <div
                  className="work__filter--text filter__text"
                >
                  widow
                </div>
                <input
                  className="checkbox"
                  name="civilState"
                  onChange={[Function]}
                  type="radio"
                  value="married"
                />
                <div
                  className="work__filter--text filter__text"
                >
                  married
                </div>
              </div>
            </div>
          </div>
          <div
            className="container__detail"
          >
            <img
              alt="sons"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/1761/1761430.svg"
            />
            <input
              className="text__detail--profile"
              onChange={[Function]}
              placeholder="sons"
              type="text"
            />
          </div>
          <div
            className="container__detail"
          >
            <img
              alt="job"
              className="icon__detail"
              src="https://image.flaticon.com/icons/svg/1063/1063196.svg"
            />
            <input
              className="checkbox"
              name="job"
              onChange={[Function]}
              type="radio"
              value="yes"
            />
            <div
              className="work__filter--text filter__text"
            >
              yes
            </div>
            <input
              className="checkbox"
              name="job"
              onChange={[Function]}
              type="radio"
              value="no"
            />
            <div
              className="work__filter--text filter__text"
            >
              no
            </div>
          </div>
        </div>
      </section>
      <section
        className="left-side__bottom"
      >
        <div
          className="image__secundary no-photo"
        >
          <input
            className="hidden"
            id="img-uploader"
            onChange={[Function]}
            type="file"
          />
          <label
            className="label"
            htmlFor="img-uploader"
          >
            Upload photos
          </label>
          0
        </div>
        <div
          className="image__secundary no-photo"
        >
          <input
            className="hidden"
            id="img2-uploader"
            onChange={[Function]}
            type="file"
          />
          <label
            className="label"
            htmlFor="img2-uploader"
          >
            Upload photos
          </label>
          0
        </div>
      </section>
    </div>
    <section
      className="right-side__detail--profile"
    >
      <div
        className="image__main--profile no-photo"
      >
        <input
          className="hidden"
          id="img3-uploader"
          onChange={[Function]}
          type="file"
        />
        <label
          className="label"
          htmlFor="img3-uploader"
        >
          Upload photos
        </label>
        0
      </div>
    </section>
    <div
      className="bottom-side"
    >
      <input
        className="presentation--profile"
        onChange={[Function]}
        placeholder="presentation"
        type="text"
      />
      <div
        className="button__container"
      >
        <a
          aria-current={null}
          href="/main"
          onClick={[Function]}
        >
          <button
            className="post-button"
            onClick={[Function]}
          >
            LOAD!
          </button>
        </a>
        <a
          aria-current={null}
          className="navLink__exit"
          href="/main"
          onClick={[Function]}
        >
          COME BACK
        </a>
      </div>
    </div>
  </form>
</main>
`;
